src/main/resources/base-target.yaml

```diff
@@ -45,6 +45,20 @@ target:
           - application/vnd.ms-excel
           - application/vnd.ms-powerpoint
           - application/vnd.openxmlformats-officedocument.presentationml.presentation
+        # The regex path patterns for the metadata ("jacket") files of binary/document files
+        metadataPathPatterns:
+          - ^/?site/documents/.+\.xml$
+        # The regex path patterns for binary/document files that are store remotely
+        remoteBinaryPathPatterns: &remoteBinaryPathPatterns
```

Comment:
Not all paths start with HTTP/HTTPS. Consider images/CSS/JS includes that simply start with `//` in order for the browser to use the current protocol in retrieving.

Reply:
Images, css and js are not indexed. The URLs in here are only for structured documents that are indexed. Binaries might not have been the best word tbh, but that's how structured documents are currently named in search classes. Should we change it?

Reply:
Wouldn't that apply to "binaries" just as well? Let's discuss tomorrow.